feature/gnoi/system/tests/copying_debug_files_test/copying_debug_files_test.go

```diff
@@ -86,9 +91,26 @@ func TestCopyingDebugFiles(t *testing.T) {
 	t.Logf("Wait 60 seconds for process to restart ...")
 	time.Sleep(60 * time.Second)
 
-	componentName := map[string]string{"name": components[dut.Vendor()]}
+	if deviations.ChassisGetRPCUnsupported(dut) {
+		var activeRp string
+		rpList := comps.FindComponentsByType(t, dut, oc.PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT_CONTROLLER_CARD)
+		t.Logf("Found RP list: %v", rpList)
+		if len(rpList) < 2 {
+			activeRp = "0/RP0/CPU0"
+		} else {
+			standbyRpName, activeRpName := comps.FindStandbyRP(t, dut, rpList)
```

Comment:
In OC, there is only CONTROLLER_CARD. RP is a cisco specific name.

```suggestion
			standbyRpName, activeRpName := comps.FindStandbyCONTROLLER_CARD(t, dut, rpList)
```